README.md

```diff
@@ -2,31 +2,31 @@
 
 # Node.js Nikita
 
-Nikita gather a set of functions usually used during system deployment.
+Nikita gathers a set of functions commonly used during system deployment.
 Documentation is available on the [project website](https://nikita.js.org).
 
 ## Main features 
 
 * Consistent Usage   
   All the functions share the same API, accepting options and a user callback in a flexible manner. Once you learn the core usage, you only learn the options of the actions you wish to execute.  
 * Everything is a file   
-  No agent to install, no database to depends on. Your project is just another Node.js package easily versionned in Git and any SCM, easily integrated with your CI and CD DevOps tools.
+  No agent to install, no database to depend on. Your project is just another Node.js package, easily versioned in Git or any SCM. Easily integrated with your CI and CD DevOps tools.
 * Idempotence   
-  Call a function multiple times and expect the same result. You’ll be informed of any modifications and can retrieve defailed information.
+  Call a function multiple times and expect the same result. You’ll be informed of any modifications and can receive detailed information.
 * Documentation
   Learn fast. Source code is self-documented with the most commons usages enriched by many examples. Don’t forget to look at the tests as well.
 * Flexibility
-  Deliberatly sacrifying speed for a maximum of strength, ease of use and flexibility. The simple API allows us to constantly add new functionnality without affecting the API.
+  Deliberately sacrifices speed for a maximum of strength, ease of use and flexibility. The simple API allows us to constantly add new functionality without affecting the API.
```

Comment:
Could you put `functionality` in plural, thanks

Reply:
Yes, ok!